### PR TITLE
Pin argo-workflows to a version before 4

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Fixes: https://github.com/thoth-station/s2i-thoth/issues/100#issue-753398656

## This introduces a breaking change

- [x] No

### CODE REVIEW

**Comment by fridex** [comment]
I will merge this to fix s2i build issues. Feel free to raise any comments.

File: Pipfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ name = "pypi"
 
 [packages]
 "rfc5424-logging-handler" = "*"
-argo-workflows = ">=3.5"
+argo-workflows = ">=3.5<4"
```
Comment (harshad16) [inline Pipfile:8]: we need a  
```suggestion
argo-workflows = ">=3.5,<4"
```
to have a version before 4. isn't @fridex ?

Comment (fridex) [reply Pipfile:8]: Yes, good catch :+1:

**Review by harshad16 — APPROVED** [review]
lgtm,
thanks :100:

**Pull request merged**